python/example_code/auto-scaling/scenario_groups_and_instances.py

```diff
@@ -292,9 +337,10 @@ def run_scenario(as_wrapper, svc_helper):
     )
     group_zones = [zones[zone_sel - 1]] if zone_sel <= len(zones) else zones
     print(f"Creating group {group_name}...")
-    as_wrapper.create_group(group_name, group_zones, template_name, 1, 1)
+    as_wrapper.create_autoscaling_group(group_name, group_zones, template_name, 1, 1)
     wait(10)
     group = as_wrapper.describe_group(group_name)
+    logger.info("Created Auto Scaling group %s.", group_name)
```

Comment:
What is the purpose of the logger statements intermingled with the print statements?